Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -5627,9 +5627,35 @@ script:
       name: device-group
     - description: The tag to filter the rules.
       name: tag
+      isArray: true
```

Comment:
If this is true, then shouldn't the description be: A comma-separated list of tags by which to filter the rules.